lib/tag-generator.js

```diff
@@ -10,20 +11,16 @@ export default class TagGenerator {
   }
 
   getPackageRoot() {
-    let packageRoot
-    if (typeof snapshotResult === 'undefined') {
-      packageRoot = path.resolve(__dirname, '..');
+    if (fs.isAbsolute(__dirname)) {
+      return path.resolve(__dirname, '..')
```

Comment:
Fixed in https://github.com/atom/symbols-view/commit/7bef6942fdae9000e3d5dceb52caca1894e4b78e. Thanks again, @maxbrunsfeld!